umap/views.py

```diff
@@ -958,20 +958,19 @@ class GZipMixin(object):
 
     @property
     def path(self):
-        return self.object.geojson.path
+        return Path(self.object.geojson.path)
 
     @property
     def gzip_path(self):
         return Path(f"{self.path}{self.EXT}")
 
-    def compute_last_modified(self, path):
-        stat = os.stat(path)
-        return http_date(stat.st_mtime)
+    def read_version(self, path):
+        return path.stem.split("_")[-1].replace(".geojson", "")
```

Comment:
```suggestion
        # Remove optional .gz, then .geojson, then return the trailing version from path.
        return path.with_suffix("").with_suffix("").split("_")[-1]
```

A non-blocking suggestion I find a bit more readable. Even with the original proposal, a comment making explicit that the `path` can be either .gz suffixed or not might be useful (it took me some time to figure it out).

Reply:
+1 for the comment and the change 👍